.gitignore

```diff
@@ -235,3 +235,4 @@ Call_of_Cthulhu_Spanish/Delta Green Variant
 /base.css
 /app.css
 /.vs/config/applicationhost.config
+WFRP4CharSheet/WFRP4Charsheet_beforebutcher.html
```

Comment:
I doubt this edit was mean to be included

Reply:
Oops, no its not. Uh, not sure how to remove the file from the submission.

Reply:
```suggestion
```
maybe this works

Reply:
If you approve the above code change the problem should be solved